pages/about/project.js

```diff
@@ -0,0 +1,33 @@
+export default function Project() {
+  return (
+    <main>
+      <h1>Project</h1>
+
+      <p>
+        Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
+        eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
+        voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
+        clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit
+        amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+        nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
+        sed diam voluptua. At vero eos et accusam et justo duo dolores et ea
+        rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem
+        ipsum dolor sit amet.
+      </p>
```

Comment:
Do we want lorem ipsum making it into the main branch? I think it is ok for developing to get a feel for the page before the actual content is available. But maybe this User Story is the place to add the content? Otherwise it would have made sense to make the other pages first and then create the links to them.

Reply:
You're totally right about that 👍
We've wrote the text for the final app to be displayed on the main.